README.md

```diff
@@ -25,22 +25,7 @@ There is a metric for the model accuracy, which is calculated by the `evaluate_m
 
 ### DSLinter
 To run only metrics from DSLinter, run:
-`pylint \
---load-plugins=dslinter \
---disable=all \
---enable=import,unnecessary-iteration-pandas,unnecessary-iteration-tensorflow,\
-nan-numpy,chain-indexing-pandas,datatype-pandas,\
-column-selection-pandas,merge-parameter-pandas,inplace-pandas,\
-dataframe-conversion-pandas,scaler-missing-scikitlearn,hyperparameters-scikitlearn,\
-hyperparameters-tensorflow,hyperparameters-pytorch,memory-release-tensorflow,\
-deterministic-pytorch,randomness-control-numpy,randomness-control-scikitlearn,\
-randomness-control-tensorflow,randomness-control-pytorch,randomness-control-dataloader-pytorch,\
-missing-mask-tensorflow,missing-mask-pytorch,tensor-array-tensorflow,\
-forward-pytorch,gradient-clear-pytorch,pipeline-not-used-scikitlearn,\
-dependent-threshold-scikitlearn,dependent-threshold-tensorflow,dependent-threshold-pytorch \
---output-format=text:report.txt,colorized \
---reports=y \
-src/`
```

Comment:
Haha, this is a nice change

Reply:
Yeah, that's basically now contained in the `.pylintrc` file